sentry-rails/spec/sentry/rails/breadcrumbs/active_support_logger_spec.rb

```diff
@@ -65,6 +65,27 @@
 
       expect(breadcrumb_buffer.count).to be_zero
     end
+
+    it "breadcrumb data only contains parameters setted by rails config" do
+      Sentry.configuration.rails.active_support_logger_subscription_items["process_action.action_controller"].delete(:controller)
+
+      get "/exception"
+
+      breadcrumbs = event.dig("breadcrumbs", "values")
+      breadcrumb = breadcrumbs.detect { |b| b["category"] == "process_action.action_controller" }
+
+      expect(breadcrumb["data"]).to include(
+        {
+          "action" => "exception",
+          "params" => { "controller" => "hello", "action" => "exception" },
+          "format" => "html",
+          "method" => "GET", "path" => "/exception",
+        }
+      )
+      expect(breadcrumb["data"].keys).not_to include("controller")
+
+      Sentry.configuration.rails.active_support_logger_subscription_items["process_action.action_controller"] << :controller
```

Comment:
You need to add this in a `ensure` block so it doesn't affect other tests if the test failed.